### PR TITLE
Update user management and payment guides

### PR DESCRIPTION
**Pull request by colder-is-better**
In light of the new rebate codes, we modify the credit card information management guide. While at it, and since there are CCMP changes related to user management, we modify the prose and refresh several screenshots in the guides on account deletion and user account management.

### CODE REVIEW

File: docs/howto/account-billing/change-credit-card.md [hunk 1]
```diff
@@ -10,16 +10,18 @@ account to switch credit cards.
...
 
-Navigate to the Dashboard list in the left-hand menu and click on _Invoices_.
+In the {{gui}}, make sure the vertical pane on the left is visible.
+Click on _Settings_, and from the drop-down menu that appears select _Manage Invoice Settings_.
 
-![White calculator and the word invoice on orange background.](assets/dashboardinvoice.png)
-
-Now you will see your invoice specifications. Click on the _circle
-with three dots_ beside the invoice you want to pay.
+Your invoice specifications are in the _Settings_ tab. To pay an invoice, click on the orange material-dots-horizontal-circle icon beside it.
```
Comment (fghaas) [inline docs/howto/account-billing/change-credit-card.md:16]: Huh? Did you mean this?
```suggestion
Your invoice specifications are in the _Settings_ tab. To pay an invoice, click on the orange :material-dots-horizontal-circle: icon beside it.
```

**Comment by colder-is-better** [comment]
> Looks good to me in general, though I do have some concerns about the edits to image alt texts.
>

I have always used short, not overly descriptive alt-texts. Sadly, I admit I have never considered the screen reader angle. I am going to restore the longer alt-text versions.
 
> Also, `docs/howto/getting-started/create-account.md` still talks about "rebate codes"; please update that.

That one escaped my attention -- thanks!

File: docs/howto/account-billing/change-account-data.md [hunk 2]
```diff
@@ -2,80 +2,77 @@
...
 
 ## Adding or removing email addresses
 
-In the _Account settings_ pane, click the middle tab, labeled _Contact_.
+In the _Account Settings_ pane, click the tab labeled _Contact_.
 This will show your currently configured email addresses.
 If there is only one email address, it will have all available roles enabled.
 
-![A 'Contact' tab with a green 'Create new Email' button to add new a email address](assets/contact.png)
+![Contact tab](assets/contact.png)
```
[the remark this answers is not in the document]
Comment (colder-is-better) [reply docs/howto/account-billing/change-account-data.md:21]: It is. I will make it not be (a regression, that is).

File: docs/howto/account-billing/change-credit-card.md (hunk 1 above)
Comment (colder-is-better) [reply docs/howto/account-billing/change-credit-card.md:16]: I am dead serious when I say that I did not.

File: docs/howto/account-billing/change-credit-card.md [hunk 3]
```diff
@@ -10,16 +10,18 @@ account to switch credit cards.
...
 
-Navigate to the Dashboard list in the left-hand menu and click on _Invoices_.
+In the {{gui}}, make sure the vertical pane on the left is visible.
+Click on _Settings_, and from the drop-down menu that appears select _Manage Invoice Settings_.
 
-![White calculator and the word invoice on orange background.](assets/dashboardinvoice.png)
-
-Now you will see your invoice specifications. Click on the _circle
-with three dots_ beside the invoice you want to pay.
+Your invoice specifications are in the _Settings_ tab. To pay an invoice, click on the orange :material-dots-horizontal-circle: icon beside it.
```
Comment (fghaas) [inline docs/howto/account-billing/change-credit-card.md:16]: Micro-nit: two sentences should use two lines.
```suggestion
Your invoice specifications are in the _Settings_ tab.
To pay an invoice, click on the orange :material-dots-horizontal-circle: icon beside it.
```

File: docs/howto/account-billing/delete-account.md [hunk 4]
```diff
@@ -1,22 +1,19 @@
...
 
-Scroll to the bottom of the pane, and click the red button named _Delete Account_.
+![Delete account tab](assets/proceed-to-deletion.png)
 
-![Black text on white background with info what will happen when you click on the delete button](assets/delete-account.png)
-
-Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_ one more time. That will create a request for account deletion.
-
-![Picture where you choose a option why you leave us. A field where you type your current password with a red delete button.](assets/reasonforleaving.png)
+Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_ one more time. That will trigger a request for account deletion.
```
Comment (fghaas) [inline docs/howto/account-billing/delete-account.md:14]:
```suggestion
Choose a reason for leaving and type in your current password.
Click the red button named _Delete Account_ one more time.
That will trigger a request for account deletion.
```

**Review by fghaas — COMMENTED** [review]
I have just two more nits, and then I'd say this is good to merge.

File: docs/howto/account-billing/change-credit-card.md (hunk 3 above)
Comment (colder-is-better) [reply docs/howto/account-billing/change-credit-card.md:16]: Ah, yes.

**Review by fghaas — APPROVED** [review]
Thanks!

**Pull request merged**